### ISSUE TITLE
Skip object via Studio

### ISSUE THREAD

**Issue by flamingotter** [bambulab/BambuStudio#3433]
**Is your feature request related to a problem? Please describe.**
No ability to skip object via Studio, compared to the ability in Handy app

**Describe the solution you'd like**
Equal functionality for skipping object in Studio as in Handy app

**Describe alternatives you've considered**
N/A

**Additional context**
None
![Screenshot_20240119-151654-EDIT](https://github.com/bambulab/BambuStudio/assets/38353250/c8bb4978-e993-44dc-a6d0-c476a1e9cd27)

**Comment by thebadluck** [comment]
I agree this feature would be very helpful in Bambu Studio.  I have a P1S so the screen is useless for most anything.   I do not always have my phone handy when I am working at my computer.  Even when I do, it takes time to open the app.  I have seen this feature requested several times.
